.github/workflows/set-release-mode.yml

```diff
@@ -0,0 +1,30 @@
+name: Set Release Mode
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release-mode:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: get version
+        id: get-product-version
+        uses: hashicorp/actions-set-product-version@v1
+
+      - name: Update version - remove -dev
+        run:  |
+          cd version
+          sed -i 's/-dev//g' 'VERSION'
```

Comment:
How will this work when we're in the prerelease phase of a release cycle? Will we need to add an input to this workflow in order to specify the actual prelease tag (say, `beta1` or `rc1`) that the script should fill in after it removes `-dev`?

Reply:
I'm going to remove the `-dev `stuff now based on our convo above. 
In terms of actual prereleases, the current behavior with the CRT version bump automation is that it will bump to the next patch version on a release branch after the release has gone out. If the version that's just been released contains a prerelease like `beta`, `alpha` etc.. it will drop this when bumping the version. So `0.4.0-alpha` would become `0.4.0` after the version bump. And then it would be up to the team to update the version if another prerelease was required. 
I think it would make sense for an input to exist in the `set-dev-mode` workflow so that the prerelease version could be manually updated after a release has gone out - to indicate the next targeted release? Alternatively this could live in the `set-release-mode` workflow if you just wanted to set this before a release. Totally up to your team here!